_posts/en/newsletters/2023-09-06-newsletter.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Bitcoin Optech Newsletter #267'
+permalink: /en/newsletters/2023/09/06/
+name: 2023-09-06-newsletter
+slug: 2023-09-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Bitcoin transaction compression:** Tom Briar [posted][briar
+  compress] to the Bitcoin-Dev mailing list a [draft specification][compress
+  spec] and [proposed implementation][compress impl] of compressed
+  Bitcoin transactions.  Smaller transactions will be more practical to
+  relay through bandwidth constrained mediums, such as by satellite or
+  through stenography (e.g., encoding a transaction in a bitmap image).
+  Traditional compression algorithms take advantage of most structured
+  data having some elements that occur more frequently than other
+  elements.  However, typical Bitcoin transactions consist largely of
```

Comment:
```suggestion
  Traditional compression algorithms take advantage of data redundancy.
  However, typical Bitcoin transactions consist largely of
```

Reply:
Is this a suggestion to make the text shorter or is it a correction?  I think the original text is more descriptive and useful, but I'd obviously want to correct any errors.